lang/rust/avro/src/encode.rs

```diff
@@ -50,24 +51,25 @@ fn encode_int(i: i32, buffer: &mut Vec<u8>) {
 /// **NOTE** This will not perform schema validation. The value is assumed to
 /// be valid with regards to the schema. Schema are needed only to guide the
 /// encoding for complex type values.
-pub fn encode_ref(value: &Value, schema: &Schema, buffer: &mut Vec<u8>) {
+pub fn encode_ref(
```

Comment:
Can we remove `pub` now ?
The wrong usage in `decode.rs` is fixed now

Reply:
Yeah I think that makes sense. On master it has the same function signature as `encode` and wasn't sure of the history there.

Reply:
I also don't know the history.